commands.go

```diff
@@ -152,13 +184,7 @@ func cmdMigrationsStatus(c *cli.Context) error {
 	}
 	log.Println(statusSeparator)
 	var last string
-	prettyWidth := 0
-	for _, each := range all {
-		pf := pretty(each.Filename)
-		if len(pf) > prettyWidth {
-			prettyWidth = len(pf)
-		}
-	}
+	prettyWidth := largestWithOf(all)
```

Comment:
Don't forget to rename this after renaming the function

Reply:
refactor tooling helps here